db-server/test/backend/db_tests.js

```diff
@@ -1761,7 +1761,7 @@ module.exports = function (config, DB) {
           })
           .then((session) => {
             // Returns verified session
-            assert.equal(session.mustVerify, null, 'mustVerify is not set')
+            assert.equal(!! session.mustVerify, false, 'mustVerify is false')
```

Comment:
Purely out of curiousity, do we actually need to keep using `!!` every time we test these properties?

`assert.equal` already uses `==` under the hood, so presumably `null` coerces to `false` anyway. And `true` is always `true`. Do any of them fail without the `!!`?

Reply:
Unfortunately, this would break some tests because the mysql column is using `TINYINT(1)` instead of `BOOLEAN`. I don't believe there is any harm in using the boolean variable since it will return true/false instead of 0,1 which needed to be coerced. Will update!

Reply:
> Unfortunately, this would break some tests because the mysql column is using TINYINT(1) instead of BOOLEAN

Turns that that was not correct and tests do pass. The only other explanation I have is that it was needed when we were using TAP. In either case, they have been updated.